R/derive_params_growth_age.R

```diff
@@ -192,7 +192,10 @@ derive_params_growth_age <- function(dataset,
   age <- assert_symbol(enexpr(age))
   age_unit <- assert_symbol(enexpr(age_unit))
   analysis_var <- assert_symbol(enexpr(analysis_var))
-  assert_data_frame(dataset, required_vars = expr_c(sex, age, age_unit, analysis_var))
+  assert_data_frame(
+    dataset,
+    required_vars = expr_c(sex, age, age_unit, analysis_var, exprs(USUBJID))
```

Comment:
we shouldn't hardcode USUBJID in functions, we need to use the more generic usual subject keys method that we always use in admiral